lib/plsql/package.rb

```diff
@@ -41,29 +47,39 @@ def procedure_defined?(name)
       PLSQL::Procedure.find(@schema, name, @package) ? true : false
     end
 
+    def [](object_name)
+      object_name = object_name.to_s
```

Comment:
I would add a downcase here, so that all 3 variants will work:
```
expect(PLSQL::Package.find(plsql, :test_package)['test_procedure']).to be_a PLSQL::Procedure
expect(PLSQL::Package.find(plsql, :test_package)['TEST_PROCEDURE']).to be_a PLSQL::Procedure
expect(PLSQL::Package.find(plsql, :test_package)[:test_procedure]).to be_a PLSQL::Procedure
```